src/StreamsAPI_Lambda/Map/ForEach.java

```diff
@@ -0,0 +1,5 @@
+package StreamsAPI_Lambda.Map;
+
+public class ForEach {
+    public static void main(String[] args) {}
```

Comment:
yes, you can iterate over a map with forEach but it looks like you have not finished this one